<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMainWindow</class>
 <widget class="QMainWindow" name="KMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtMuic音乐播放器-王政</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #eeeeee;
    height: 8px;
    background: #eeeeee;
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: #d81e06;  /* 红色滑动条 */
    border: 1px solid #eeeeee;
    width: 14px;
    margin: -2px 0;
    border-radius: 4px;
}

QSlider::groove:vertical {
    border: 1px solid #eeeeee;
    width: 8px;
    background: #eeeeee;
    margin: 0 2px;
}

QSlider::handle:vertical {
    background: #d81e06;  /* 红色滑动条 */
    border: 1px solid #eeeeee;
    height: 14px;
    margin: 0 -2px;
    border-radius: 4px;
}

QPushButton {
    background-color: #eeeeee; 
    border: 1px #000000;
    border-radius: 0px;
    padding: 0px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">    background-color: #FFFFFF;  /* 深灰色背景 */
</string>
   </property>
   <widget class="QSlider" name="m_playingPercentHorizontalSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>610</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeHorCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string extracomment="鼠标拖动调节进度"/>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="m_controlWidget" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>315</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent; 
    border: none;
    border-radius: 0px;
    padding: 0px;
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_lastItemPushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/originalLeftButton.png</normaloff>image/originalLeftButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_startPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/originalPlayButton.png</normaloff>image/originalPlayButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_nextItemPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/originalRightButton.png</normaloff>image/originalRightButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>480</y>
      <width>51</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="m_voiceVerticalSlider">
       <property name="cursor">
        <cursorShape>SizeVerCursor</cursorShape>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_voiceControlPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>40</horstretch>
         <verstretch>40</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/soundIcon.png</normaloff>image/soundIcon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>231</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="m_localtab">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>本地曲库</string>
     </attribute>
     <widget class="QTableView" name="m_localTableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>335</width>
        <height>393</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="m_onlinetab">
     <attribute name="title">
      <string>在线曲库</string>
     </attribute>
     <widget class="QTableView" name="m_onlineTableView2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>335</width>
        <height>393</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="m_currSongLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
font: 11pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>QtMusic播放器</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="m_currDurationLabel">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>372</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_totalDurationLabel">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="m_picLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>378</width>
      <height>378</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  min-width: 366px;
  min-height: 366px;
  max-width: 366px;
  max-height: 366px;
  border-radius: 183px; 
  border: 6px solid #d81e06;
}

color:#d81e06</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>好听音乐马上来</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="m_voiceLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>610</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>音量:20</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_songWordsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 14pt &quot;Arial&quot; 700;
font-weight: bold;
color: #d81e06;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="m_onlineTableView">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>351</width>
      <height>541</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView::item:selected { background-color: #d81e06;color:#ffffff }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_playModePushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>510</y>
      <width>49</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>image/sortPlay.png</normaloff>image/sortPlay.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="m_playModLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>60</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>顺序播放</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_logolabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/musicLogo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>540</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>鼠标双击播放</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
